java/ql/test/utils/modelgenerator/dataflow/p/MultipleImpl2.java

```diff
@@ -0,0 +1,20 @@
+package p;
+
+class MultipleImpl2 {
```

Comment:
I think it would be helpful for future readers of the code to add a comment explaining what intended behaviour this test is testing.